editors-support.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Editors support
```

Comment:
s/Editors/Editor/ support. Not sure certain which grammar rule this is (indefinite plurals?), but the noun goes in singular form.